mylyn.builds/org.eclipse.mylyn.hudson.core/src/org/eclipse/mylyn/internal/hudson/core/client/HudsonOperation.java

```diff
@@ -109,14 +160,15 @@ protected void authenticate(IOperationMonitor monitor) throws IOException {
 
 			// success
 			getClient().setAuthenticated(hasValidatAuthenticationState());
+
+			updateCrumb(monitor);
 		} finally {
 			HttpUtil.release(request, response, monitor);
 		}
-
-		updateCrumb(monitor);
 	}
 
-	public HttpResponse executeAuthenticationRequest(IOperationMonitor monitor, UserCredentials credentials,
+	@Deprecated
```

Comment:
Please drop.

Reply:
Alas, the Hudson/Jenkins servers used for the unit tests doesn't support the new way, so I put them back